src/tangerine/resources/assistant.py

```diff
@@ -12,12 +12,23 @@
 from tangerine.embeddings import embed_query
 from tangerine.models.assistant import Assistant
 from tangerine.models.interactions import store_interaction
-from tangerine.search import search_engine
+from tangerine.search import SearchResult, search_engine
 from tangerine.utils import File, add_filenames_to_assistant, embed_files, remove_files
 from tangerine.vector import vector_db
 
 log = logging.getLogger("tangerine.resources")
 
+MODELS = {
+    "default": {
+        "base_url": config.LLM_BASE_URL,
+        "name": config.LLM_MODEL_NAME,
+        "api_key": config.LLM_API_KEY,
```

Comment:
Once again here - if we have multiple models, we will very likely want multiple API keys and multiple base urls.

Reply:
Is config just env vars?